charts/factorio/values.yaml

```diff
@@ -68,6 +68,11 @@ service:
   # externalTrafficPolicy: Cluster
   annotations: {}
 
+#### Deployment Configuration ####
+## @section Deployment Parameters
+## @param deployment.hostPort Specify the hostPort you want to use for the game (default port 34179 is recommended)
+# deployment:
+#   hostPort: 34179
```

Comment:
I commentend the values out, to be able to check if the value is set or not. 

Don't know if you are happy with it. I could also use an "enabled" key or something like this.

Reply:
What you had before was more correct. `hostNetworking: true` (or false) is what will enable/disable host config. It's `hostPort` that isn't needed. I think having `hostNetworking: false` by default is best.

Reply:
Ah, alright. I'm not that deep into Kubernetes networking, so if you think hostNetwork is the better option to use, then let's go with that. I'll change that in a minute.

Reply:
Well, it's not that it's a better option. It's that HostNetworking is the toggle-able thing, and hostNetwork isn't really a thing.